src/test/mir-opt/generator-tiny/rustc.main-{{closure}}.generator_resume.0.mir

```diff
@@ -1,17 +1,13 @@
 // MIR for `main::{{closure}}#0` 0 generator_resume
 /* generator_layout = GeneratorLayout {
-    field_tys: {
-        _0: HasDrop,
```

Comment:
Should we be concerned about this change?

Reply:
Nope. Discussion from the last PR is here: https://github.com/rust-lang/rust/pull/71946#discussion_r423237651